bHY6yh-d56217/cod-bxxqqq_bHY6yh-d56217.xml

```diff
@@ -0,0 +1,268 @@
+              in se Crimen habet quanto
+              <lb ed="#B"/>maior qui peccat habetur</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e441"> 
+              <lb ed="#B"/><ref xml:id="bHY6yh-d56217-Rd1e501">libro 5 et 
+              <lb ed="#B"/>finali satura 2</ref> <quote xml:id="bHY6yh-d56217-Qd1e443">velocius et citius nos 
+              <lb ed="#B"/>corrumpunt viciorum exempla domesti<!--line jumps down-->ca</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e451">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d56217-Rd1e516">PAladius de agricultura capitulo 
+              <lb ed="#B"/>I</ref> <quote xml:id="bHY6yh-d56217-Qd1e455">quod est prohemium Nos quos 
+              <lb ed="#B"/>reprehendimus immitemur</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e461">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d56217-Rd1e529">CAto capitulo 4</ref> <quote xml:id="bHY6yh-d56217-Qd1e465">Multorum disce<!--line jumps up--> exem<!--line jumps down-->plo que 
+              <lb ed="#B"/>facta sequaris ¶ Que fugias 
+              <lb ed="#B"/>vita nobis est aliena magistra</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e474">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d56217-Rd1e543">CAssiodorus epistolarum libro 5 epistola 44 et 
+              <lb ed="#B"/>finale</ref> <quote xml:id="bHY6yh-d56217-Qd1e478">Instructus redditur animus in 
+              <lb ed="#B"/>futuris quoniam preteritorum commonetur 
+              <lb ed="#B"/>exemplis</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e486">
+              ¶ <ref xml:id="bHY6yh-d56217-Rd1e555">libro 10 epistola 13</ref> <quote xml:id="bHY6yh-d56217-Qd1e490">Res minor 
+              <lb ed="#B"/>ad potioris currit exemplum et 
+              <lb ed="#B"/>alieni facti iure inuidiam sub<lb ed="#B" break="no"/>stinet
+              qui peccatis prestat exemplum</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e498"> 
+              <lb ed="#B"/>¶ <ref xml:id="bHY6yh-d56217-Rd1e570">libro 12 et finale epistola I</ref> <quote xml:id="bHY6yh-d56217-Qd1e500">Sicud discipuli 
+              <lb ed="#B"/>magistri scientiam produnt sic nos 
+              <lb ed="#B"/>obsequentium mores aperiunt</quote>
+            </p>
+            <p xml:id="bHY6yh-d56217-d1e508">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d56217-Rd1e585">CLaudianus libro I</ref> <quote xml:id="bHY6yh-d56217-Qd1e512">Componitur orbis
+              <lb ed="#B"/>regis ad exemplum nec sic in<lb ed="#B" break="no"/>flectere 
+              sensus / Humanos edic<lb ed="#B" break="no"/>ta. 
+              valent quam vita regentis / 
+              <lb ed="#B"/>¶ Mobile mutatur semper cum princi<lb ed="#B" break="no"/>pe 
+              <cb ed="#B" n="b"/>
+              uulgus</quote>
```

Comment:
I'm unclear on what's happening here.

Does the word break over the column, if so it should be: 

`princi<cb ed="#B" n="b"/><lb ed="#B" break="no"/>pe`